packages/python/readme_metrics/MetricsApiConfig.py

```diff
@@ -29,25 +29,30 @@ class MetricsApiConfig:
         If you configure a blacklist, it will override any whitelist configuration.
     WHITELIST: List[str]
         (optional) An array of headers and JSON body properties to send to ReadMe.
+    ALLOWED_HTTP_HOSTS: List[str]
+        (optional) A list of allowed http hosts for sending data to the ReadMe API.
 
         If this option is configured, ONLY the whitelisted properties will be sent.
     """
     README_API_KEY: str = None
     BUFFER_LENGTH: int = 10
-    GROUPING_FUNCTION = lambda req: None
+    GROUPING_FUNCTION: Callable[[Any], None] = lambda req: None
```

Comment:
Thanks for this, I couldn't figure out how to satisfy PEP8 with this default arg.